egs/gigaspeech/ASR/RESULTS.md

```diff
@@ -1,4 +1,75 @@
 ## Results
+### zipformer (zipformer + pruned stateless transducer)
+
+See <https://github.com/k2-fsa/icefall/pull/1254> for more details.
+
+[zipformer](./zipformer)
+
+- Non-streaming
+- normal-scaled model, number of model parameters: 65549011, i.e., 65.55 M
+
+You can find a pretrained model, training logs, decoding logs, and decoding results at:
+<https://huggingface.co/yfyeung/icefall-asr-gigaspeech-zipformer-2023-10-17>
```

Comment:
Could you upload the tensorboard log to
https://wandb.ai/site
and post a link to it here?

Reply:
OK.

Reply:
By the way, the link has not been posted. please leave a message when you think this PR is ready to merge.

Reply:
https://wandb.ai/yifanyeung/icefall-asr-gigaspeech-zipformer-2023-10-20

Reply:
OK, I have posted that.